powerdns_recursor/datadog_checks/powerdns_recursor/powerdns_recursor.py

```diff
@@ -126,6 +126,27 @@ def check(self, instance):
                     self.gauge('powerdns.recursor.{}'.format(stat['name']), float(stat['value']), tags=tags)
                 elif stat['name'] in PowerDNSRecursorCheck.RATE_METRICS_V4:
                     self.rate('powerdns.recursor.{}'.format(stat['name']), float(stat['value']), tags=tags)
+        self._collect_metadata(config)
+
+    def _collect_metadata(self, config):
```

Comment:
(Nit) Should we move `_collect_metadata()` to the end of the class body? Metadata collection is "non-essential", and having it "front and center" like this makes it a bit harder to understand what the "essential" steps in the check execution are.